market/connector/docusign/demo/eSignatureDemoTest/src_test/net/docusign/test/TestJwtBuilder.java

```diff
@@ -0,0 +1,74 @@
+package net.docusign.test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Method;
+import java.lang.reflect.Proxy;
+import java.security.PrivateKey;
+import java.util.Properties;
+
+import javax.ws.rs.core.Configuration;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Test;
+
+import ch.ivyteam.ivy.rest.client.FeatureConfig;
+import net.docusign.auth.JwtFactory;
+import net.docusign.auth.OAuth2Feature.Property;
+
+public class TestJwtBuilder
+{
+  @Test
+  void build()
+  {
+    Properties p = new Properties();
+    p.setProperty(Property.AUTH_BASE_URI, "https://account-d.docusign.com/oauth");
+    p.setProperty(Property.CLIENT_ID, "24adbf4f-");
+    p.setProperty(Property.SYSTEM_USER_ID, "ee1e53b6-");
+    p.setProperty(Property.SCOPE, "signature impersonation");
+    FeatureConfig config = new FeatureConfig(configMock(p), TestJwtBuilder.class);
+    
+    String jwtToken = new JwtFactory(config).createToken();
+    assertThat(jwtToken).isNotNull();
+    System.out.println(jwtToken);
+  }
+  
+  @Test
+  void pemRead() throws IOException
+  {
+    // generator:
+    // ssh-keygen -t rsa -b 2048 -m PKCS8 -f testKey.pem
```

Comment:
is this working on windows too?
maybe write generator on linux?

Reply:
I think no: but it's just a hint for devs how the test key was generated.   
nobody has to actually do it today, since I've added the generated output as file.
Still, since I had hard times to generate the key exactly like 'docusign' does, I'd love to document the taken approach.